### DISCUSSION TITLE
QueryStakeDelegDeposits Ledger State Query

### DISCUSSION THREAD

**Discussion by mirceahasegan** [CardanoSolutions/ogmios#359] [Q & A] [answered]
Hello,
I couldn't find an equivalent Ledger State Query for (QueryStakeDelegDeposits](https://github.com/IntersectMBO/cardano-api/blob/f4a380cd763a27354f1d76798a59c6e1635d8bdd/cardano-api/internal/Cardano/Api/Query.hs#L298-L300).
Is there one?
If not, is there any plan to add it?
Thank you!

**Comment by KtorZ** [comment, marked as answer]
Correct; there are a few new queries related to the Conway era such as this one. I am still waiting for the Conway era to fully stabilize before adding support for them. So this will likely come in a future release; unless there's already a strong demand for them.
